test/integration/scheduler_perf/util.go

```diff
@@ -282,3 +323,51 @@ func (tc *throughputCollector) collect() []DataItem {
 
 	return []DataItem{throughputSummary}
 }
+
+// getExtensionPointDurationHist return a labeld Histogram for `scheduler_framework_extension_point_duration_seconds` metric.
+// TODO: Consolidate this method with `testutil.GetHistogramFromGatherer` to remove the duplicated code.
+func getExtensionPointDurationHist(gatherer metrics.Gatherer, metricName string) ([]labeledHist, error) {
```

Comment:
since we're using this func with only scheduler_framework_extension_point_duration_seconds, do we need the second argument?

Reply:
removed.